### ISSUE TITLE
Data file to drive banner

### ISSUE THREAD

**Issue by geekgonecrazy** [RocketChat/rocketchat.github.io#343]
We have a lot of PR's to add and remove the html for banners.  Happens enough we should have driven from a data file

**Comment by isabellarussell** [comment]
Agreed! Not quite sure how the technical side of this works @geekgonecrazy but definitely agree to finding a solution to avoid the constant PRs :) Who is best placed to help with this?

**Comment by geekgonecrazy** [comment]
I think creating a PR still works.  But might be useful to be able to add events ahead of time and a bit of javascript or something to handle the displaying of the banners when they should be?  That way we can just add new events and they will automatically rotate out.  Or maybe even if multiple happening at same time could alternate.

**Comment by isabellarussell** [comment]
Definitely! @danilowoz do you think you could tackle this for us?
